plugin-api/pom.xml

```diff
@@ -29,6 +29,10 @@
     <artifactId>plugin-api</artifactId>
 
     <dependencies>
+        <dependency>
+            <groupId>com.google.guava</groupId>
+            <artifactId>guava</artifactId>
+        </dependency>
```

Comment:
Why use this library? If you are only using its utility package, there is no need to introduce it. Please remove it.

Reply:
com.megaease.easeagent:core dependency com.google.guava:guava. 
use com.google.common.base.Splitter utility for parse  otel.resource.attributes(eg, key1=value1,key2=value).

Reply:
> com.megaease.easeagent:core dependency com.google.guava:guava. use com.google.common.base.Splitter utility for parse otel.resource.attributes(eg, key1=value1,key2=value).

If you are using Guava in the subproject 'config,' you should include the JAR file there. Please avoid using it in the 'plugin-api' project.